### PR TITLE
Handle CQL Embedded in Questionnaire

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR handles the CQL embedded in Questionnaire.

To test:

Switch to branch embedCql in repo CRD, DTR and CDS-Library
For regular form, open HomeOxygenThreapy order form, verify the last field "Form Expiration Date" is prepopulated with a date 2 weeks from the day you open the form.
For adaptive form, open HomeOxygenTherapy "additional" form, click the "Next Question" to open the form, verify the field "Form Expiration Date" is prepopulated with a date 2 weeks from the day you open the form.

### CODE REVIEW

**Review by plarocque4 — COMMENTED** [review]
Looks good.

**Review by plarocque4 — APPROVED** [review]
Maybe get @kghoreshi to take a look before pushing it up.

File: src/App.jsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ export default class App extends Component {
           this.setState({ questionnaire: artifacts.questionnaire });
           this.setState({ deviceRequest: deviceRequest });
           this.setState({ isAdaptiveFormWithoutExtension: artifacts.questionnaire.meta && artifacts.questionnaire.meta.profile && artifacts.questionnaire.meta.profile.includes("http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-adapt") && (artifacts.questionnaire.extension === undefined || !artifacts.questionnaire.extension.includes(e => e.url === "http://hl7.org/fhir/StructureDefinition/cqf-library")) });
-          
+          this.setState({ });
```
Comment (KeeyanGhoreshi) [inline src/App.jsx:132]: Was this supposed to be left in?

**Comment by KeeyanGhoreshi** [comment]
It works for the date but I could not get any information to prefill with the embedded cql.  Is there information about this feature anywhere, like the IG or in a readme?

**Comment by yuliumitre** [comment]
> 
@KeeyanGhoreshi can you elaborate "I could not get any information to prefill with the embedded cql" pls? Are you expecting pre-population statement like CQL included in a Library? 

Here is the HL7 JIRA ticket:   https://jira.hl7.org/browse/FHIR-33224 

Specifically, this paragraph mentions CQL embedded in the Questionnaire: 

The DTR app SHALL support loading and rendering the adaptive form developed by the payer following the SDC adaptive form workflow. If there is CQL embedded in the questionnaire or the associated library, the DTR app should be able to execute the CQL based on the questions loaded in the questionnaire.

**Comment by KeeyanGhoreshi** [comment]
If I wanted to prefill the patients name with embedded CQL, could I do that?

**Comment by KeeyanGhoreshi** [comment]
Ok, in that case this works as expected

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**